vue-frontend/src/components/portfolio/DesignSection.vue

```diff
@@ -24,18 +24,15 @@
   <section class="image tw-relative tw-z-[-1]">
     <aspect-ratio height="56.25%" class="tw-overflow-hidden">
       <img
-        v-if="response[0].responsiveImages"
         :src="response[0].backgroundImage[3]"
-        :srcset="`${response[0].backgroundImage[0]} 400w, ${response[0].backgroundImage[1]} 800w, ${response[0].backgroundImage[2]} 1200w, ${response[0].backgroundImage[3]} 1600w`"
-        sizes="(min-width: 992px) 45vw, 100vw"
-        class="tw-w-full tw-h-full tw-object-cover"
+        :srcset="`${response[0].backgroundImage[0]} 400w, ${response[0].backgroundImage[1]} 800w,${response[0].backgroundImage[2]} 1400w,${response[0].backgroundImage[3]} 2400w`"
+        sizes="
+        (min-width: 2420px) 2200px, 
+    (min-width: 720px) calc(94.76vw - 274px), 
```

Comment:
Can you explain this calculation?

Reply:
its width of the image across all three breakpoints.Actually old sizes have rendering problem so i have used this code and its working fine.